docs/source/cpp/tables.rst

```diff
@@ -77,6 +77,17 @@ has a schema which must match its arrays' datatypes.
 Record batches are a convenient unit of work for various serialization
 and computation functions, possibly incremental.
 
+.. image:: tables-versus-record-batches.svg
+   :alt: A graphical representation of an Arrow Table and a Record Batch, with
+         structure as described in text above.
+
+Because record batches can be represented as a struct array, they can be 
+exported through the C data interface between implementations. Tables and 
```

Comment:
Do we want to assume knowledge of the C Data Interface?

We could make this more generic: "they can be sent between implementations, such as via IPC (link), or via the C Data Interface (link)"

Reply:
Also, the glossary talks about this as well, dunno if that's useful

Reply:
Also I wouldn't go too much into details here, and remove the mention about "can be represented as a struct array".